src/dialog-service.ts

```diff
@@ -185,7 +186,7 @@ export class DialogService {
         }
         return Promise.reject(reason);
       });
-    })).then((unclosedControllers) => unclosedControllers.filter(unclosed => !!unclosed));
+    })).then(unclosedControllers => unclosedControllers.filter(unclosed => !!unclosed));
```

Comment:
Yes. When a dialog successfully closes - `undefined` is returned, those that were cancelled return the controller itself. The end result should be and Promise<DialogController[]> - the controllers of the dialogs that were cancelled.